meta-flex-distro/conf/distro/flex-os.conf

```diff
@@ -17,7 +17,7 @@ QUARTER_VERSION = "04"
 UPDATE_VERSION = "0"
 RELEASE_VERSION = "${YEAR_VERSION}.${QUARTER_VERSION}.${UPDATE_VERSION}"
 DISTRO_VERSION = "${RELEASE_VERSION}"
-ARCHIVE_RELEASE_VERSION = "${DISTRO_VERSION}.${BSP_VERSION}.${PATCH_VERSION}"
+ARCHIVE_RELEASE_VERSION = "${DISTRO_VERSION}"
```

Comment:
I think we can also remove these variables now:
BSP_VERSION
PATCH_VERSION

Reply:
removed it from the script